sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/MetricFeedbackFilter.Serialization.cs

```diff
@@ -45,5 +45,12 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
             }
             writer.WriteEndObject();
         }
+
+        internal static RequestContent ToRequestContent(MetricFeedbackFilter model)
+        {
+            var content = new Utf8JsonRequestContent();
```

Comment:
Should we check for model == null here?

Reply:
We are adding all the validations in the method [here](https://github.com/ShivangiReja/azure-sdk-for-net/blob/MetricsAdvisor-Experiment/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs#L165-L186) before calling ToRequestContent(model) method and if we pass null in `GetAllFeedbackAsync` method, then it throws `ArgumentNullException`.

Here is a test case validating [this](https://github.com/ShivangiReja/azure-sdk-for-net/blob/MetricsAdvisor-Experiment/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorClient/MetricFeedbackTests.cs#L76).